bears/markdown/MarkdownBear.py

```diff
@@ -30,7 +30,7 @@ class MarkdownBear:
 
     _output_regex = re.compile(
         r'\s*(?P<line>\d+):(?P<column>\d+)'
-        r'\s*(?P<severity>warning)\s*(?P<message>.*)')
+        r'\s*(?P<severity>warning)\s*(?P<message>.+?)(?:  .*|\n|$)')
```

Comment:
Actually I would rephrase the shortlog to something like `Improve result message` or so, as this is the effective reason :)